cmd/main.go

```diff
@@ -2,13 +2,45 @@ package main
 
 import (
 	"fmt"
+	"github.com/hashgraph/hedera-sdk-go"
+	hederasdk "github.com/limechain/hedera-eth-bridge-validator/app/clients/hedera"
+	http "github.com/limechain/hedera-eth-bridge-validator/app/clients/http"
 	"github.com/limechain/hedera-eth-bridge-validator/app/persistence"
+	consensus_message "github.com/limechain/hedera-eth-bridge-validator/app/process/watcher/consensus-message"
+	crypto_transfer "github.com/limechain/hedera-eth-bridge-validator/app/process/watcher/crypto-transfer"
 	"github.com/limechain/hedera-eth-bridge-validator/config"
 	"github.com/limechain/hedera-watcher-sdk/server"
+	"log"
 )
 
 func main() {
 	configuration := config.LoadConfig()
 	persistence.RunDb(configuration.Hedera.Validator.Db)
-	server.NewServer().Run(fmt.Sprintf(":%s", configuration.Hedera.Validator.Port))
+	server := server.NewServer()
+
+	mirrorNodeClient, e := hederasdk.NewMirrorClient(configuration.Hedera.MirrorNode)
```

Comment:
We should be creating a new `hedera` client and not use the `hedera.NewMirrorClient`. I would actually think about creating the instance of the client inside the watchers and not globally here. At some point the client might have a PK to sign transactions. In that case, we would like to create an instance of the hedera client with a given PK and that client would be used only by the cryptotransfer handler that should post consensus messages